apps/zui/electron-builder.json

```diff
@@ -10,11 +10,11 @@
     "sign": "./scripts/sign.js"
   },
   "linux": {"target": ["deb", "rpm"]},
+  "mac": {"entitlements": "darwin.plist", "notarize": {"teamId": "2DBXHXV7KJ"}},
```

Comment:
The `teamId` value here is redundant with the value in the `APPLE_TEAM_ID` environment variable, but it's needed here due to open issue https://github.com/electron-userland/electron-builder/issues/8103. I'm Subscribed to that issue so I can revisit this if/when it gets fixed. Also, [found consensus](https://stackoverflow.com/questions/44417171/should-i-keep-the-apple-developer-team-id-secret) validated my guess that this team ID does not seem to be sensitive info (e.g., similar to a company/domain name), so I've gone ahead and put the value here rather than adding yet more code to populate it from a Secret.